default.nix

```diff
@@ -1,15 +1,37 @@
 { system ? builtins.currentSystem
 , pkgs ? import (import ./nix/nixpkgs.nix) { inherit system; }
 }:
-let
-  # Small src cleaner.
-  source = import ./nix/source.nix;
-in
 rec {
-  devshell = pkgs.callPackage ./nix/devshell.nix { inherit source; };
-  devshell-docs = pkgs.callPackage ./nix/mdbook.nix { inherit source; };
-  mkDevShell = pkgs.callPackage ./nix/mkDevShell.nix { };
+  # CLI
+  cli = pkgs.callPackage ./devshell { };
 
-  # This project's shell
-  devshell-shell = mkDevShell.fromTOML ./devshell.toml;
+  # Docs
+  docs = pkgs.callPackage ./docs { };
+
+  # Evaluate the devshell module
+  eval = import ./modules pkgs;
+
+  # Loads a Nix module from TOML
+  importTOML = file:
+    let
+      dir = builtins.dirOf file;
+      data = builtins.fromTOML (builtins.readFile file);
+    in
+    {
+      _file = file;
+      imports = map (str: "${toString dir}/${str}") (data.imports or [ ]);
+      config = builtins.removeAttrs data [ "imports" ];
```

Comment:
So it's very clear how to provide a relative import, however I wonder how we could import a selection out of a hypothetical flake `devshellModules.ruby` attribute which itself would represent a devshell module?

Reply:
This is currently unhandled and the user would have to revert to writing Nix code.
```nix
devshell.build {
  import = [
    devshellModules.ruby
    (devshell.fromTOML ./devshell.toml)
  ];
}
```